tests/unit/components/ContentNode.spec.js

```diff
@@ -112,29 +112,6 @@ describe('ContentNode', () => {
       expect(codeListing.props('content')).toEqual(listing.code);
       expect(codeListing.isEmpty()).toBe(true);
     });
-
-    it('renders a `Figure`/`Figcaption` with metadata', () => {
```

Comment:
Why was this test removed?

Reply:
I thought it was duplicated but in fact it wasn't 😅 I put it back!